modules/application/src/main/java/edu/northeastern/truthtree/dto/StoryDTOWrapper.java

```diff
@@ -0,0 +1,42 @@
+package edu.northeastern.truthtree.dto;
```

Comment:
This seems redundant, why not just a `Map<String, Object>` to parse the response instead of a DTO wrapper? (Refer back to my comment in `StoriesAssembler.java`)

Reply:
Easier way to get a dto mapped. realStoryDTO.setContent((String) bodyMap.get("content")) is just one field. Same thing needs to be added for all the fields. I was following the same approach but then in the response, depending upon the timestamp values length,  I was getting int or long which became difficult to typecast. I implemented this approach only but realized, keeping a wrapper dto will make code simpler.

Reply:
Refer this commit which follows the changes you mentioned https://github.com/TruthTreeASD/backend/commit/e0f5bba2a27512544f5bc04d807d56d7e04d0452

Reply:
Yea I actually prefer your previous commit. Doing that vs. creating a wrapper class are essentially the same thing: i.e. you specify/typecast each field to the respective types. Just an advantage of using map is that you have 1 less class to worry about. Just my 2 cents